javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,79 @@
+//Create array of bad movie and movies under 2000
+function fetchMovie() {
+  fetch(
+    'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json'
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      const badMovies = data.filter((movie) => {
+        return movie.rating < 5;
+      });
+      console.log(badMovies);
+      const badMovies2000 = badMovies.filter((movie) => {
+        return movie.year < 2000;
+      });
+      console.log(badMovies2000);
```

Comment:
Instead of doing the filter again you can write the condition to separate the movies since year 2000 (i.e movies after 2000), and i would suggest to use async await instead for fetching the data.
 ( .then((data) => {
            const badMovies2000 = [];
            const badMovies = data.filter((movie) => {
                if (movie.year > 2000 && movie.rating < 5) {
                    badMovies2000.push(movie)
                    return;
                }
                return movie.rating < 5;
            });
            console.log(badMovies);
            console.log(badMovies2000);
        });)